testdata/license-status.jsonnet

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
Please add a comment in the first row of what URL one can use to collect the API request from. See the other jsonnet files for example.